### DISCUSSION TITLE
How to call the scripts unattended?

### DISCUSSION THREAD

**Discussion by alexandre-leites** [tteck/Proxmox#1659] [Help Needed] [answered]
I would like to possibly setup my entire Proxmox LXCs and VMs unattended. It is possible to call the scripts unattended with prespecified configurations, or only the default ones? Therefore, no UI questions will be asked.

**Comment by tteck** [comment, marked as answer]
The current script configuration prevents unattended installations, and there are no plans to include this feature in the future.

**Reply by alexandre-leites** [reply]
Do you have a reason for keeping that way, or it is just the way it is because of time constraint? I am asking because If I get time to do that, then I can try to make a PR

**Reply by tteck** [reply]
It's my preference.
